### ISSUE TITLE
Incorrect input values of mouse event position with `{plotnine}`

### ISSUE THREAD

**Issue by youngroklee-ml** [posit-dev/py-shiny#1007]
When getting mouse event on `ui.output_plot()`, e.g., `click`, `brush`, etc., the input value accessed in the server does not correctly represents user's input on a plot.

For example, a snapshot below shows that I dragged a region for `[2, 3]` on x-axis and `[20, 30]` on y-axis, but the input values were read as `[1.7, 2.9]` for x-axis and `[21.9, 30.6]` for y-axis.

<img width="986" alt="plotnine-brush-position" src="https://github.com/posit-dev/py-shiny/assets/30917347/f5b185f5-cec7-44d4-a9a1-5adce94f1913">

Below is an app code:

```
from shiny import App, render, ui, req
from plotnine import ggplot, aes, geom_point
from pydataset import data

mtcars = data("mtcars")

app_ui = ui.page_fluid(
    ui.output_plot("plot", brush=True),
    ui.output_text_verbatim("info"),
)

def server(input, output, session):
    @render.plot
    def plot():
        res = (ggplot(mtcars, aes('wt', 'mpg'))
               + geom_point())
        return res
    
    @render.text
    def info():
        req(input.plot_brush())
        return input.plot_brush()

app = App(app_ui, server)
```

**Comment by wch** [comment]
I believe this is the same as https://github.com/has2k1/plotnine/issues/738, and should be fixed by #999.

**Comment by wch** [comment]
This has been fixed by #999.
